File: Minecraft Plugin/src/br/com/minecart/MineCart.java
package br.com.minecart;

import java.io.File;
import java.sql.SQLException;
import java.util.logging.Level;

import org.bukkit.Bukkit;
import org.bukkit.configuration.file.YamlConfiguration;
import org.bukkit.plugin.java.JavaPlugin;

import br.com.minecart.commands.MainCommand;
import br.com.minecart.database.Database;

public class MineCart extends JavaPlugin {
	public YamlConfiguration ResourceMessage;
	public MineCartKeyManage keysManage;
	public Database database;
	
	public static MineCart instance;
	
	public String MineCartAutorization;
	public String MineCartServerToken;
	
	public final String MineCartAPI = "https://minecart.com.br/api";
	
	public void onEnable(){
		instance = this;
		
		if(!this.setupDatabase()){
			Bukkit.getPluginManager().disablePlugin(this);
			
			return;
		}
		
		this.keysManage = new MineCartKeyManage();
		
		if(!new File(getDataFolder(), "config.yml").exists()) saveDefaultConfig();
		
		this.loadMessages();
		
		this.MineCartAutorization = getConfig().getString("MineCart.ShopKey", "");
		this.MineCartServerToken = getConfig().getString("MineCart.ShopServerKey", "");
		
		getCommand("resgatarvip").setExecutor(new MainCommand());
		getCommand("resgatarcash").setExecutor(new MainCommand());
		getCommand("minhaskeys").setExecutor(new MainCommand());
		getCommand("ativar").setExecutor(new MainCommand());
		getCommand("minecart").setExecutor(new MainCommand());
	}
	
	public static Database getDB() {
		return instance.database;
	}
	
    private boolean setupDatabase(){
        try {
            database = new Database(getConfig().getConfigurationSection("database"));
        } catch (ClassNotFoundException exception){
            getLogger().log(Level.SEVERE, String.format("Unable to register JDCB driver: %s", exception.getMessage()));
            return false;
        } catch (SQLException exception){
            getLogger().log(Level.SEVERE, String.format("Unable to connect to SQL server: %s", exception.getMessage()));
            return false;
        }

        try {
            database.createTables();
        } catch (Exception exception){
            getLogger().log(Level.SEVERE, String.format("An exception was thrown while attempting to create tables: %s", exception.getMessage()));
            return false;
        }

        return true;
    }
    
    private void loadMessages(){
    	File resourceMessage = new File(getDataFolder(), "messages.yml");
    	
    	if(!resourceMessage.exists()) saveResource("messages.yml", false);
    	
    	this.ResourceMessage = YamlConfiguration.loadConfiguration(resourceMessage);
    }
}

File: Minecraft Plugin/Minecraft Bedrock/src/Minecart/Minecart.php
<?php

namespace Minecart;

use Minecart\commands\Redeem;
use pocketmine\plugin\PluginBase;
use pocketmine\utils\Config;

use Minecart\commands\MyKeys;
use Minecart\utils\Configuration;
use Minecart\utils\Messages;

class Minecart extends PluginBase
{
    public $messages = [];
    public $config  = [];

    public $cooldown = [];

	public static $instance;

    public function onEnable() : void
	{
		$this->registerInstance();
		$this->registerCommands();
		$this->registerConfig();
		$this->registerMessages();

		$this->getServer()->getLogger()->info("§7Plugin §aMinecart§7 ativado com sucesso!");
	}

	public function registerCommands() : void
	{
		$this->getServer()->getCommandMap()->register('mykeys', new MyKeys());
        $this->getServer()->getCommandMap()->register('redeem', new Redeem());
	}

	public function registerInstance() : void
	{
		self::$instance = $this;
	}

	public static function getInstance() : Minecart
	{
		return self::$instance;
	}

    public function registerMessages() : void
    {
        $this->saveResource("messages.yml");
        $messages = new Config($this->getDataFolder() . "messages.yml", Config::YAML);
        $this->messages = $messages->getAll();
	}

    public function registerConfig() : void
    {
        $this->saveResource("config.yml");
        $config = new Config($this->getDataFolder() . "config.yml", Config::YAML);
        $this->config = $config->getAll();
	}

    public function getMessage(string $key) : string
    {
        return Messages::getMessage($key);
    }

    public function getCfg(string $key) : string
    {
        return Configuration::getConfig($key);
    }
}